crates/bevy_audio/src/audio.rs

```diff
@@ -128,6 +128,8 @@ pub struct PlaybackSettings {
     pub repeat: bool,
     /// Volume to play at.
     pub volume: f32,
+    /// Ignore global volume
+    pub absolue_volume: bool,
```

Comment:
Typo here

Reply:
```suggestion
    pub absolute_volume: bool,
```

This will need a rename elsewhere though.